cmd/zetaclientd/debug.go

```diff
@@ -70,36 +72,39 @@ func DebugCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
 			chainParams, err := client.GetChainParams()
 			if err != nil {
 				return err
 			}
+
 			tssEthAddress, err := client.GetEthTssAddress()
 			if err != nil {
 				return err
 			}
-			chain := chains.GetChainFromChainID(chainID)
+
+			chain := chains.GetChainFromChainID(chainID, coreContext.GetAdditionalChains())
```

Comment:
The term `AdditionalChains` confused me a bit at first glance. Do you mean something like a 'ChainBook' or 'ChainDictory'?